fabric-api-lookup-api-v1/src/testmod/java/net/fabricmc/fabric/test/lookup/compat/WrappedInventory.java

```diff
@@ -61,7 +61,7 @@ public ItemStack tryInsert(ItemStack input, boolean simulate) {
 				ItemStack stack = inv.getStack(i);
 
 				if (stack.isEmpty() || ItemStack.canCombine(stack, input)) {
-					int remainingSpace = Math.min(inv.getMaxCountPerStack(), stack.getItem().getMaxCount()) - stack.getCount();
+					int remainingSpace = Math.min(inv.getMaxCountPerStack(), stack.getItem().getMaxStackSize(stack)) - stack.getCount();
```

Comment:
```suggestion
					int remainingSpace = Math.min(inv.getMaxCountPerStack(), stack.getMaxCount()) - stack.getCount();
```
I'd prefer calling the stack-aware method on the stack rather than the item, as it's less verbose